components/Cmip6MonthlyChartControls.vue

```diff
@@ -67,6 +67,39 @@ chartStore.labels = {
   },
 }
 
+// Sea ice concentration data is only available for some models.
+if (props.datasetKeys?.includes('siconc')) {
+  chartStore.labels = {
+    models: {
+      'HadGEM3-GC31-LL': 'HadGEM3-GC31-LL',
+      'HadGEM3-GC31-MM': 'HadGEM3-GC31-MM',
+      MIROC6: 'MIROC6',
+      'NorESM2-MM': 'NorESM2-MM',
+      TaiESM1: 'TaiESM1',
+    },
+    scenarios: {
```

Comment:
This key, and months, are already defined in the object that is being overridden so I don't think they need to be redeclared here!

Reply:
I've modified this to use the spread operator to only modify the object variable models.